### ISSUE TITLE
Hermit shell hooks emits error

### ISSUE THREAD

**Issue by mightyguava** [cashapp/hermit#246]
When I use hermit's shellhooks in zsh, in a hermit enabled repo, I get the following error at the end of every single command I run.

```
fatal:hermit: /Users/yunchi/Projects/bork/bin/hermit has an unknown SHA256 signature (9413f2347c5f70e6a004e62b7faac99d3bb1666f86451ed1f6e05a679e3bc27c); verify that you trust this environment and run 'hermit init /Users/yunchi/Projects/bork'
```

This doesn't happen if I disable the shell hook and activate hermit manually.

**Comment by mightyguava** [comment]
Ah, so if I run `. bin/activate-hermit` everything is fine.

But if I run `eval "$(hermit shell-hooks --print --zsh)"` I get these errors.

**Comment by alecthomas** [comment]
This should be fixed in the latest release. `hermit update`

**Comment by alecthomas** [comment]
Actually I don't see that signature at all, can you attach the bin/ directory as a zip file? And what does `hermit --version` say?

**Comment by alecthomas** [comment]
Actually nm ,was able to replicate. Should be fixed in 0.18.2 rolling out soon.

**Comment by mightyguava** [comment]
Yay it works! Magic!
